app/components/post_textbox/post_textbox_base.js

```diff
@@ -407,17 +408,27 @@ export default class PostTextBoxBase extends PureComponent {
         }
     };
 
-    handlePostDraftSelectionChanged = (event) => {
-        const cursorPosition = event.nativeEvent.selection.end;
+    handleOnSelectionChange = (e) => {
```

Comment:
Rename `e` to `event` to align with the other function definitions.... easier to search in the codebase too.